lucetc/lucetc/options.rs

```diff
@@ -251,6 +268,13 @@ impl Options {
                     .multiple(false)
                     .help("output destination, defaults to a.out if unspecified"),
             )
+            .arg(
+                Arg::with_name("target")
+                    .long("target")
+                    .takes_value(true)
+                    .multiple(false)
+                    .help("target to compile for, defaults to host if unspecified"),
```

Comment:
Could we print the actual value (`Triple::host()`) here, so that people get a sense of what is expected for the `--target` value?